lib/Quantization/Base/Base.cpp

```diff
@@ -262,5 +262,30 @@ std::vector<int8_t> createMapping(TypeRef inTy, TypeRef outTy,
   return mapping;
 }
 
+void tensorRowwiseQuantization(Tensor &input, Tensor &qData, Tensor &scales,
```

Comment:
Nits:
- `output` would probably be clearer than `qData`.
- Can `input` be `const Tensor&`?

Reply:
If we put input as const Tensor&, we can't use "getHandle()" function : "'this' argument has type 'const glow::Tensor', but method is not marked const
template <class ElemTy> Handle<ElemTy> Tensor::getHandle()" .

Although we can use "getUnsafePtr()" method for const Tensor, the code will be complicated...

Reply:
I think it would make sense to add one more method
```
template <class ElemTy = float> const Handle<ElemTy> getHandle() const;
```
this should resolve the issue you are seeing.

Reply:
@rdzhabarov In that case, I think the class Handle need to be improved as well. The current constructor is `explicit Handle(Tensor *tensor) : tensor_(tensor)` , and the private member `tensor` is not const. So I think if we decide to support getHandle() for const Tensor,  it is better to create a separate repo :)